src/screens/HomeScreen.js

```diff
@@ -115,76 +120,89 @@ export const HomeScreen = ({ navigation }) => {
-                      },
-                      details: newsItem
-                    },
-                    bottomDivider: index !== data.newsItems.length - 1,
-                    __typename: newsItem.__typename
-                  }));
+                    const newsItems =
+                      data &&
+                      data.newsItems &&
+                      data.newsItems.map((newsItem, index) => ({
+                        id: newsItem.id,
+                        subtitle: subtitle(
+                          momentFormat(newsItem.publishedAt),
+                          !!newsItem.dataProvider && newsItem.dataProvider.name
+                        ),
+                        title:
+                          !!newsItem.contentBlocks &&
+                          !!newsItem.contentBlocks.length &&
+                          newsItem.contentBlocks[0].title,
+                        routeName: 'Detail',
+                        params: {
+                          title: categoryTitleDetail,
+                          query: QUERY_TYPES.NEWS_ITEM,
+                          queryVariables: { id: `${newsItem.id}` },
+                          rootRouteName: 'NewsItems',
+                          shareContent: {
+                            message: shareMessage(newsItem, QUERY_TYPES.NEWS_ITEM)
+                          },
+                          details: newsItem
+                        },
+                        bottomDivider: index !== data.newsItems.length - 1,
+                        __typename: newsItem.__typename
+                      }));
 
-                if (!newsItems || !newsItems.length) return null;
+                    if (!newsItems || !newsItems.length) return null;
 
-                return (
-                  <View>
-                    <TextList navigation={navigation} data={newsItems} />
+                    return (
+                      <View>
+                        <TextList navigation={navigation} data={newsItems} />
 
-                    <Wrapper>
-                      <Button
-                        title={buttonNews}
-                        onPress={() => navigation.navigate(NAVIGATION.NEWS_ITEMS_INDEX)}
-                      />
-                    </Wrapper>
-                  </View>
-                );
-              }}
-            </Query>
-          </>
-        )}
+                        <Wrapper>
+                          <Button
+                            title={categoryButton}
+                            onPress={() =>
+                              navigation.navigate(
+                                NAVIGATION.NEWS_ITEMS_INDEX({ categoryId, categoryTitle })
+                              )
+                            }
+                          />
+                        </Wrapper>
+                      </View>
+                    );
+                  }}
+                </Query>
+              </Fragment>
+            )
+          )}
 
```

Comment:
How is this result on the screen connected with this PR ? <img width="839" alt="Screenshot 2020-11-09 at 13 00 28" src="https://user-images.githubusercontent.com/43641321/98539002-e86a2100-228b-11eb-9690-864e6952b8dc.png">

Reply:
this screenshot shows the separation of newsitems into two category blocks "nachrichten" and "informationen". before there were all newsitems in one block. now there are some newsitems in the first and others in the second.